docs/en/stack/ml/df-analytics/evaluatedf-api.asciidoc

```diff
@@ -0,0 +1,100 @@
+[role="xpack"]
+[[ml-dfanalytics-evaluate]]
+== Evaluating {dfanalytics}
+
+Using the {dfanalytics} features to gain insights from a data set is an 
+iterative process. You might need to experiment with different analyses, 
+parameters, ways to transform data before you arrive to a result that satisfies 
+your use case. A valuable companion to this process is the 
+{ref}/evaluate-dfanalytics.html[{evaluatedf-api}] that allows you to evaluate 
+the {dfanalytics} performance against a marked up data set. It helps to 
+understand error distributions and to know the points where the {dfanalytics} 
+model performing well or less trustworthily.
+
+{evaluatedf-api-cap} is designed for providing a general evaluation mechanism 
+for the different kinds of {dfanalytics}. That means that every type of analysis 
+can be evaluated by using one of the evaluation types that the {evaluatedf-api} 
+offers. For example, the results of an {oldetection} analysis can be evaluated 
+by using {binarysc} through the {evaluatedf-api}.
+
+To evaluate an algorithm with this API, you need to annotate your index that 
```

Comment:
> To evaluate an algorithm...

It stands out that we're talking about evaluating an algorithm whereas up to now we've been talking about evaluating data frame analytics.  Is the change here important?